portfolio/src/main/webapp/script.js

```diff
@@ -13,20 +13,28 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings = [
-    'I play tennis.',
-    '"An office is a place where dreams come true." — Michael Scott (The Office).',
-    '"A bug is never just a mistake. It represents something bigger. An error of thinking that makes you who you are." - Elliot (Mr Robot).',
-    'I have three dogs and one cat.'
-  ];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+* Insert/remove the class in the element.
+*/
+function toggleClass(element, classAttribute) {
+  if (!element || typeof element.className !== "string" || !classAttribute || typeof classAttribute !== "string") {
+    return;
+  }
+  const classes = element.className.split(" ");
+  if (classes.indexOf(classAttribute) === -1) {
+    classes.push(classAttribute);
+  } else {
+    classes.splice(classes.indexOf(classAttribute), 1);
+  }
+  element.className = classes.join(" ");
+}
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+/**
+ * Show/hide the description inside the section.
+ */
+function toggleDescription(sectionId) {
+  if (!document.getElementById(sectionId)) {
+    return;
+  }
+  const sectionElement = document.getElementById(sectionId);
```

Comment:
if you move this statement to the top, you only have to call `document.getElementById()` once.

https://refactoring.guru/smells/duplicate-code